ipn/ipnlocal/peerapi.go

```diff
@@ -1124,17 +1163,43 @@ func (h *peerAPIHandler) handleServeTailFS(w http.ResponseWriter, r *http.Reques
 
 	p, err := tailfs.ParsePermissions(rawPerms)
 	if err != nil {
+		h.logf("tailfs: error parsing permissions: %w", err.Error())
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
 	fs, ok := h.ps.b.sys.TailFSForRemote.GetOK()
 	if !ok {
-		http.Error(w, "tailfs not enabled", http.StatusNotFound)
+		h.logf("tailfs: not supported on platform")
+		http.Error(w, "tailfs not supported on platform", http.StatusNotFound)
 		return
 	}
+	wr := &httpResponseWrapper{
+		ResponseWriter: w,
+	}
+	bw := &requestBodyWrapper{
+		ReadCloser: r.Body,
+	}
+	r.Body = bw
+
+	defer func() {
+		if r.Method != httpm.PUT && r.Method != httpm.GET {
+			return
+		}
+
+		switch wr.statusCode {
+		case 304:
+			// 304s are particularly chatty so skip logging.
+		default:
+			fileExt := filepath.Ext(r.URL.Path)
```

Comment:
Files don't always have an extension. Why don't we log files without an extension?

Reply:
We can, it will just get very chatty. Will amend.

Reply:
Interesting. I guess this condition happened to exclude all of those hidden files that the OS creates and uses?

Reply:
Yeah, I think so. But really - I while working with this, I didn't see the value in including those lines in general. I could be wrong in my assessment, but that is what lead me to having that conditional.